build-tools/scripts/DotNet.targets

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <Target Name="PackDotNet">
+    <PropertyGroup>
+      <_TopDir>$(MSBuildThisFileDirectory)..\..\</_TopDir>
+    </PropertyGroup>
+    <MSBuild Projects="$(_TopDir)build-tools\xa-prep-tasks\xa-prep-tasks.csproj" />
+    <MSBuild Projects="$(_TopDir)Xamarin.Android.sln" Properties="DisableApiCompatibilityCheck=true" />
+    <MSBuild Projects="$(_TopDir)build-tools\create-packs\Microsoft.Android.Sdk.proj" Targets="CreateAllPacks" />
+    <!-- Clean up old, previously restored packages -->
+    <ItemGroup>
+      <_OldPackages Include="$(_TopDir)packages\microsoft.android.*\**\*.nupkg" />
+      <_DirectoriesToRemove Include="%(_OldPackages.RootDir)%(_OldPackages.Directory)" />
+    </ItemGroup>
+    <RemoveDir Directories="@(_DirectoriesToRemove)" />
+  </Target>
```

Comment:
This looks great. I had a similar helper target in a branch while working on the Mono.Android tests, and it may be a good candidate for this as well? It generates `nuget.config` and `global.json` which point to the newly built packages. If that seems useful we could add this target and output these files to `.\bin\BuildDebug\nupkgs\`. Otherwise I will add it in a future PR. The target is here:
https://github.com/pjcollins/xamarin-android/commit/bb1dd6df7c8e5525c22b2f2f348992b19f0bf96e#diff-6fc2685fd76642c79c8ad32367445f42R6-R48

Reply:
`build-tools\scripts\DotNet.targets` might be a good place to put `GenerateAndroidSdkProjectConfigs`.

But maybe we save it for later for when it is needed?

Reply:
I'm happy to add it later 👍